apps/core-app/src/legos/tx.ts

```diff
@@ -0,0 +1,28 @@
+import { POSTER_TAGS } from '@daohaus/common-utilities';
+import { buildMultiCallTX } from '@daohaus/tx-builder-feature';
+
+import { CONTRACT } from './contracts';
+
+export const TX = {
+  POST_SIGNAL: buildMultiCallTX({
+    id: 'POST_SIGNAL',
+    actions: [
+      {
+        contract: CONTRACT.POSTER,
+        method: 'post',
+        args: [
+          {
+            type: 'JSONDetails',
+            jsonSchema: {
+              title: `.formValues.title`,
+              description: `.formValues.description`,
+              link: `.formValues.link`,
+              proposalType: { type: 'static', value: 'Signal Proposal' },
+            },
+          },
+          { type: 'static', value: POSTER_TAGS.daoProfileUpdate },
```

Comment:
Just realized that this isn't a good idea. We'll need a tag for this.

Reply:
ok yeah and we need an agreed upon schema. let think about that.

just title, description and link? we'll also need daoid in there.

Reply:
and we'll need to do the subgraph stuff - but good to get some work around that.

let's talk later on it